doc/source/myrocks/zenfs.rst

```diff
@@ -115,11 +205,23 @@ Shut down the server and use the following command to backup a ZenFS file system
 
 .. note::
 
-    At this time, a logical backup is the only backup type supported by the Percona Server MyRocks tables located on ``ZenFS``.
+    At this time, a logical backup is the only backup type supported by the *Percona Server for MySQL*. The MyRocks tables are located on ``ZenFS``.
 
 Use the following command to restore a backup:
 
 .. sourcecode:: bash
 
     zenfs restore --zbd=nvme1n2 --path=/storage/backup/rocksdb-backup \
     --restore_path=./
+
+			
+Known Limitations
+====================================================================
+
+After a reboot the NVME ZBD configuration ("/dev/nvme02" in our examples) may not reappear. The issue may be OS-dependent and may be managed by the system admins. One or more of the following events may have occurred:
+
+- A reboot changes the active "scheduler" from "[mq-deadline]"
+
+- A reboot resets the device permissions from "640/mysql:mysql" to "660/root:disk".
```

Comment:
Can't we ask our experts on how to make these changes permanently? There must be ways to remove these limitations.

Reply:
As @AgileStas suggested, device ownership can also be changed at boot time via `udev`
https://www.thegeekdiary.com/how-to-configure-device-file-owner-group-with-udev-rules/

Reply:
Updated.